src/createAgentManager.ts

```diff
@@ -47,10 +47,17 @@ function getAgentStreamArgs(agent: Agent): CreateStreamOptions {
             stream_warmup: true,
         };
     }
+    let config
+    if(agent.presenter.type === VideoType.Talk && agent.presenter.stitch) {
+        config = {
+            stitch: agent.presenter.stitch
+        }
+    }
     return {
         videoType: VideoType.Talk,
         source_url: agent.presenter.source_url,
         stream_warmup: true,
+        config: config && Object.keys(config).length > 0 ? config : undefined
```

Comment:
Make this to not send config object when stitch is false, to avoid any possible issue inside create stream endpoint